procurement_group_carrier/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Camptocamp
+# Copyright 2020-2021 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Procurement Group Carrier",
+    "Summary": "Record the carrier on the procurement group",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "category": "Warehouse Management",
+    "depends": ["sale_stock", "delivery"],
```

Comment:
I would have named it sale_procurement_group_carrier or you need to put sale related stuff out of this.

Reply:
I agree

Reply:
Then sale_stock_procurement_group_carrier ?
It is more a stock feature than a sale one.
There is already a module "sale_stock_mto_as_mts_orderpoint", so i think that will be more in line with the naming convention.

Reply:
The carrier is from the delivery module (that depends on sale_stock).
So I'm naming it delivery_procurement_group_carrier !